packages/foam-vscode/src/services/vscode-watcher.ts

```diff
@@ -0,0 +1,29 @@
+import { IDisposable, Emitter, URI, IDataStoreWatcher } from "foam-core";
+import { FileSystemWatcher } from "vscode";
+
+export class VscodeBaseWatcher implements IDataStoreWatcher, IDisposable {
```

Comment:
Nitpick but shouldn't it be `VSCodeBaseWatcher`?

Reply:
close, was double checking the vscode codebase to see what convention they use, and it's `VsCode`, so we could call this `VsCodeBaseWatcher`